rs_inverse/scripts/gui_spot.py

```diff
@@ -24,6 +22,8 @@ def __init__(self, win, pub):
         self.lbl11 = Label(win, text='ClearanceHeight')
```

Comment:
all these variables should have meaningful names, as they are used in different places